package.json

```diff
@@ -34,7 +43,7 @@
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "git://github.com/germanrcuriel/jira-cmd.git"
+    "url": "git://github.com/palashkulsh/jira-cmd.git"
```

Comment:
Probably you should leave this as what it was

Reply:
yes, my bad. will correct it

Reply:
done!!